OpenHPL/Icons/DraftTube.mo

```diff
@@ -0,0 +1,41 @@
+within OpenHPL.Icons;
+partial class DraftTube "Draft tube icon"
+  parameter Boolean vertical=false "Display vertical icon instead"
+    annotation(Dialog(group = "Icon"),
+    choices(checkBox = true));
```

Comment:
What is the point of this unused parameter?

Reply:
I do not know actually. I duplicated icon of pipe made by Luibomyr. 
(The mesh-grid coloring is also not supported in slanted icons like drafttube)

Reply:
Well I know what it is for in the Icon you copied it from (I implemented that) but it does no sense here. So be careful when copying over code blindly. You should always know what the code does before using it :smirk:

Reply:
Thanks. I hope to not repeat again.